modules/end-user-guide/pages/using-git-credentials.adoc

```diff
@@ -0,0 +1,9 @@
+:navtitle: Using Git credentials
```

Comment:
Why is this a separate file? Wouldn't it be simpler to have a single procedure file with instructions for both the store and the token? Or, if those should be separate procedures, then keep them separate, but make a regular assembly out of this file.

Reply:
But i guess i drop those references.

Reply:
OK, so that's what i'll do.

Reply:
Done. Thank you again for this suggestion.